build.gradle

```diff
@@ -222,7 +222,7 @@ project(':cruise-control') {
     compile 'io.dropwizard.metrics:metrics-core:3.2.6'
     compile 'com.nimbusds:nimbus-jose-jwt:8.5'
     compile 'com.101tec:zkclient:0.11'
-    compile group: 'io.swagger.parser.v3', name: 'swagger-parser', version: '2.0.18'
+    compile group: 'io.swagger.parser.v3', name: 'swagger-parser-v3', version: '2.0.18'
```

Comment:
I think we should bump this to 2.0.19 too, so we get the latest.

Reply:
I agree, bumped the version.